### PR TITLE
Amazing features now work!

### PR DESCRIPTION
**Pull request by Hunter-aw**
i WROKED REALLY HARD ON THIS AMAZING FEATURE THAT REALLY WORKS
and kicks ass

### CODE REVIEW

File: dist/main.js [hunk 1]
```diff
@@ -0,0 +1 @@
+var lol = "haha!!!"
```
Comment (Hunter-aw) [inline dist/main.js:1]: this is ridiculous!

**Review by SivanYad — COMMENTED** [review]
Dear avinoam please use let instead of var

File: dist/main.js (hunk 1 above)
Comment (avinoamMO) [reply dist/main.js:1]: I quit!

Comment (danielkar1) [reply dist/main.js:1]: bugs are pepole !

**Pull request merged**